conans/client/tools/win.py

```diff
@@ -442,6 +442,11 @@ def vcvars_command(conanfile=None, arch=None, compiler_version=None, force=False
                        'x86_64': 'x86_amd64',
                        'armv7': 'x86_arm',
                        'armv8': 'x86_arm64'}.get(arch_setting)
+    elif arch_build == 'armv8':
```

Comment:
This is modifying only the legacy tool ``from conans``
This should be done as a secondary thing, as a backport, but this should be addressed first in ``from conan`` space.

Also some unittest would be necessary, please ask for guidance if you need it.
Thanks for contributing to Conan!